OpenRA.Mods.Common/Traits/World/Pathfinder/PathSearch.cs

```diff
@@ -0,0 +1,137 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using OpenRA.Primitives;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	using System.Linq;
+
+	public sealed class PathSearch : BasePathSearch
+	{
+		public override IEnumerable<Pair<CPos, int>> Considered
+		{
+			get { return considered; }
+		}
+
+		LinkedList<Pair<CPos, int>> considered;
+
+		#region Constructors
```

Comment:
They have help me greatly when scanning the code to ease readability. I would not touch them if possible...